core/src/main/java/jenkins/model/NewViewLink.java

```diff
@@ -14,7 +14,7 @@
     @VisibleForTesting
     static final String ICON_FILE_NAME = "folder";
     @VisibleForTesting
-    public static final String URL_NAME = "newView";
+    static final String URL_NAME_SUFFIX = "newView";
```

Comment:
formally it is a compat-breaking change. I would recommend not to do so

Reply:
Should probably have been `@Restricted` to begin with, AFAIUI `@VisibleForTesting` serves the same purpose, just not enforcing it.

Reply:
There's no reason the entire class shouldn't be `@Restricted`.